### PR TITLE
Fixes broken windows tests

### PR DESCRIPTION
**Pull request by TwitchBronBron**
Fixes a few issues causing unit tests on windows to fail. The auto-imports using wrong path separators also meant the framework itself wouldn't produce correct results on windows.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,13 +104,14 @@
         "tag-pattern": ".+"
     },
     "scripts": {
-        "clean": "rm -rf ./dist && mkdir dist",
+        "clean": "rimraf ./dist && mkdir dist",
```
Comment (TwitchBronBron) [inline package.json:107]: windows doesn't have `rm` command, so use `rimraf` node module which works cross-platform

File: package.json [hunk 2]
```diff
@@ -102,13 +104,14 @@
...
         "compile": "npm run clean && tsc -p .",
         "prepublishOnly": "npm run build",
-        "build": "npm run compile && cp -r src/lib/framework ./dist/lib/framework",
+        "build": "npm run compile && cpx \"src/lib/framework/**/*\" ./dist/lib/framework",
```
Comment (TwitchBronBron) [inline package.json:110]: windows doesn't have `cp` command, so use `cpx` node module which works cross platform.

File: package.json [hunk 3]
```diff
@@ -102,13 +104,14 @@
...
         "lint": "eslint \"src/**/*.ts\"",
         "cli": "npm run compile && npm unlink && npm link",
-        "test": "npm run build && nyc mocha src/**/*.spec.ts --full-trace --bail",
+        "test": "nyc mocha",
```
Comment (TwitchBronBron) [inline package.json:113]: All of these settings are already present in the `"mocha"` package.json section, they don't need to be duplicated here.

File: package.json [hunk 4]
```diff
@@ -102,13 +104,14 @@
...
+        "test:nocover": "mocha",
```
Comment (TwitchBronBron) [inline package.json:114]: When running unit tests in vscode, it's faster to run the tests without coverage, so the `"test"` task in vscode is now bound to this non-coverage npm script.

File: src/lib/Utils.ts [hunk 5]
```diff
@@ -273,5 +273,17 @@ export function addImport(file: BrsFile, pkgPath: string) {
         // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
         file.attachDependencyGraph(dg);
     }
+}
 
+/**
+ * Handles replacing windows `\` with `/`. Also handles v0->v1 compatibility by prepending `pkg:/` if missing
+ * @param pkgPath the full pkgPath of a file (with or without `pkg:/`). This MUST NOT be a relative path
+ */
+export function sanitizePkgPath(pkgPath: string) {
```
Comment (TwitchBronBron) [inline src/lib/Utils.ts:282]: bsc@v1 stores the full `pkg:/...` path in the `file.pkgPath` variable, but bsc@v0 stores it without the `pkg:/` and with os-specific path separators (idk why I did it this way....but oh well). 

This function handles sanitizing the pkgPath to always produce a proper `pkg:/` path with the correct `/` paths. However, you can't use this function when you're going to compare `file.pkgPath`, so only use this function right before writing pkgPaths to Roku files.

**Review by georgejecook — APPROVED** [review]

**Pull request merged**